packages/api-explorer/style/ErrorBoundary.scss

```diff
@@ -0,0 +1,35 @@
+.hub-reference-error {
```

Comment:
cc @rafegoldberg can you take a look through this?

Reply:
@rafegoldberg wrote this lol

Reply:
haha, but come to mention it... I should autoprefix this manually (since we've only added PostCSS to readme proper, as far as I know...) Pushing an update now.